src/lib/server/auth/jsonwebtoken/jws/lib/tostring.ts

```
@@ -0,0 +1,7 @@
+import { Buffer } from 'node:buffer'
+
+export const toString = (obj: unknown): string => {
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)